.goreleaser.yml

```diff
@@ -11,14 +11,14 @@ builds:
     ldflags:
       - -s -w -X main.version=v{{.Version}}
 
-archive:
-  format: tar.gz
-  format_overrides:
-    - goos: windows
-      format: zip
-  replacements:
-    amd64: x86_64
-    386: x86_32
-    darwin: osx
-  files:
-    - LICENSE
+archives:
+  - format: tar.gz
+    format_overrides:
+      - goos: windows
+        format: zip
+    replacements:
+      amd64: x86_64
+      386: x86_32
+      darwin: osx
+    files:
+      - LICENSE
```

Comment:
This whole thing shouldn't be left aligned?  I thought the way arrays worked in yaml, this could be a 1 character change.

Reply:
You are right: the leading `-` of an array can be at the same level as its parent key. But I was maintaining indentation consistency with other arrays in the file (including in this very stanza), which all add one level of indentation for the `-`.